helm/cas-ciip-portal/templates/cronJobs/cron-load-testing-data.yaml

```diff
@@ -0,0 +1,74 @@
+{{- if hasSuffix "-dev" .Release.Namespace }}
+apiVersion: batch/v1beta1
+kind: CronJob
+metadata:
+  name: {{ template "cas-ciip-portal.fullname" . }}-load-testing-data
+  labels:
+{{ include "cas-ciip-portal.labels" . | indent 4 }}
+  annotations:
+    "helm.sh/hook": "pre-upgrade,pre-install"
+    "helm.sh/hook-delete-policy": "before-hook-creation"
+    "helm.sh/hook-weight": "-5"
+spec:
+  suspend: true # This cron job is intended to be triggered manually
+  schedule: "* * * * *"
+  jobTemplate:
+    spec:
+      backoffLimit: 0
+      template:
+        metadata:
+          labels:
+{{ include "cas-ciip-portal.labels" . | indent 14 }}
+        spec:
+          activeDeadlineSeconds: 1800
+          restartPolicy: Never
+          containers:
+            - env:
+                - name: SQITCH_TARGET
+                  value: "db:pg:"
+                - name: PGUSER
+                  valueFrom:
+                    secretKeyRef:
+                      key: database-user
+                      name: {{ template "cas-ciip-portal.fullname" . }}
+                - name: PGPASSWORD
+                  valueFrom:
+                    secretKeyRef:
+                      key: database-password
+                      name: {{ template "cas-ciip-portal.fullname" . }}
+                - name: PGDATABASE
+                  valueFrom:
+                    secretKeyRef:
+                      key: database-name
+                      name: {{ template "cas-ciip-portal.fullname" . }}
+                - name: PGPORT
+                  value: "5432"
+                - name: PGHOST
+                  value: {{ template "cas-ciip-portal.fullname" . }}-patroni
+                - name: SQITCH_FULLNAME
+                  value: OpenShift Deployer
+                - name: SQITCH_EMAIL
+                  value: deployer@pathfinder.gov.bc.ca
+                - name: NAMESPACE
+                  valueFrom:
+                    fieldRef:
+                      apiVersion: v1
+                      fieldPath: metadata.namespace
+              name: {{ template "cas-ciip-portal.fullname" . }}-load-testing-data
+              image: {{ .Values.image.schema.repository }}:{{ .Values.image.schema.tag }}
+              resources:
+                limits:
+                  cpu: 500m
+                  memory: 128Mi
+                requests:
+                  cpu: 100m
+                  memory: 64Mi
+              command:
+                - /usr/bin/env
+                - bash
+                - -c
+                - |
+                  set -euo pipefail;
+                  schema/data/deploy-data.sh --oc-project=$(NAMESPACE);
```

Comment:
```suggestion
                  schema/data/deploy-data.sh --oc-project=$(NAMESPACE) --swrs-load-testing;
```
?

Reply:
This shouldn't need any flags, if we deploy the load-testing data in cas-ggircs & then redeploy the ciip data it should pull the load testing data from the cas-ggircs swrs database. `--swrs-load-testing` is for local development if we wanted to go look at it locally